includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -278,6 +280,14 @@ protected function get_row_data( $item ) {
 
 		switch ( $this->type ) {
 			case 'courses':
+				// Last Activity.
+				$last_activity_date = 'N/A';
```

Comment:
Should this be localized? It's kind of an abbreviation so I'm not sure.

Reply:
Yeah, probably there's a case to be made here for not using abbreviations, but I've at least localized it for now. https://github.com/Automattic/sensei/pull/4879/commits/dec1456b3508583a39fab4b08848277ec34bc58d